apps/lottie-preview/src/locations/ConfigScreen.tsx

```diff
@@ -1,41 +1,63 @@
 import { ConfigAppSDK } from '@contentful/app-sdk';
-import { Heading, Paragraph, Autocomplete, Flex, Checkbox, Pill, TextLink, FormLabel, Card, Text, Note } from '@contentful/f36-components';
+import { Heading, Paragraph, Autocomplete, Flex, Checkbox, Pill, TextLink, FormLabel, Card, Text, Note, Subheading } from '@contentful/f36-components';
 import tokens from '@contentful/f36-tokens';
 import { useSDK } from '@contentful/react-apps-toolkit';
 import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
 import { styles } from './ConfigScreen.styles';
-import { ExternalLinkIcon } from '@contentful/f36-icons';
+import { ClockIcon, ExternalLinkIcon } from '@contentful/f36-icons';
 import { useJsonFieldsState } from '@src/hooks/useJsonFieldsState';
-import { buildEditorInterfaceControls, getJsonFields, groupFieldsByContentType } from '@src/configUtils';
+import { getJsonFields, getContentTypesWithJsonFieldsCount, groupFieldsByContentType, JsonFieldsResult, JsonField } from '@src/configUtils';
+
+const AppWidgetNamespace = 'app';
+const DefaultWidgetId = 'objectEditor';
```

Comment:
[nit pick] Capitalizing the first letter on these variables makes them look more like typescript types to me. I think `UPPER_SNAKE_CASE` or `lowerCamelCase` are more standard for constants in typescript